modules/installation-creating-gcp-worker.adoc

```diff
@@ -0,0 +1,87 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_gcp_user_infra/installing-gcp-user-infra.adoc
+
+[id="installation-creating-gcp-worker_{context}"]
+= Creating additional worker machines in GCP
+
+You can create worker machines in Google Cloud Platform (GCP) for your cluster
+to use by launching individual instances discretely or by automated processes
+outside the cluster, such as Auto Scaling Groups. You can also take advantage of
+the built-in cluster scaling mechanisms and the machine API in {product-title}.
+
+In this example, you will manually launch one instance using the Deployment Manager
+template. Additional instances can be launched by including additional
```

Comment:
I think that this should be an optional step after you create the template file.